pennylane/pauli/pauli_vspace.py

```diff
@@ -164,15 +164,14 @@ def add(self, other, tol=None):
 
         We can generate a ``PauliVSpace`` and add a linearly independent operator to its basis.
 
-        >>> ops = [X(0), X(1)]
+        >>> ops = [qml.X(0), qml.X(1)]
         >>> vspace = qml.pauli.PauliVSpace(ops)
```

Comment:
Just curious, why only in this example the class from this module (`PauliVSpace`) gets called with `qml.pauli`?

For e.g., L217 below doesn't have it; and similarly in the module `pauli_arithmetic`, the docstrings for `PauliWord` and `PauliSentence` didn't have it (see e.g. L162 in `pauli_arithmetic`)